lizmap/modules/admin/controllers/landing_page_content.classic.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Lizmap administration : landing page content.
+ *
+ * @author    3liz
+ * @copyright 2016 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+class landing_page_contentCtrl extends jController
+{
+    // Configure access via jacl2 rights management
+    public $pluginParams = array(
+        '*' => array('jacl2.right' => 'lizmap.admin.access'),
+    );
+
+    /**
+     * Display a wysiwyg editor.
+     */
+    public function index()
+    {
+        $rep = $this->getResponse('html');
+
+        // Get HTML content from ini file
+        $lizmapConfigIniFile = jApp::configPath('lizmapConfig.ini.php');
+        $iniModifier = new jIniFileModifier ($lizmapConfigIniFile);
+
```

Comment:
no, you must not use jIniFileModifier to read an ini file. It takes too much resources.

And you should not read lizmapConfig.ini.php directly, use the class LizmapService